crates/rpc/rpc-builder/tests/it/http.rs

```diff
@@ -73,6 +75,10 @@ where
     EthApiClient::block_transaction_count_by_hash(client, hash).await.unwrap();
     EthApiClient::block_uncles_count_by_hash(client, hash).await.unwrap();
     EthApiClient::block_uncles_count_by_number(client, block_number).await.unwrap();
+    // using temp unwrap_or_default because errors aren't getting handled here
+    EthApiClient::transaction_by_hash(client, tx_hash).await.unwrap_or_default();
+    EthApiClient::transaction_by_block_hash_and_index(client, hash, index).await.unwrap_or_default();
+    EthApiClient::transaction_by_block_number_and_index(client, block_number, index).await.unwrap_or_default();
```

Comment:
Since this is a test, and the call should not error, `unwrap` should be preferred here over `unwrap_or_default`